consular/main.py

```diff
@@ -318,9 +320,20 @@ def register_task_service(self, app_id, task_id, host, port):
             The ID of the task, this will be used as the Consul service ID.
         :param str host:
             The host address of the machine the task is running on.
-        :param int port:
-            The port number the task can be accessed on on the host machine.
+        :param list ports:
+            The port numbers the task can be accessed on on the host machine.
         """
+        if not ports:
+            port = None
+        elif len(ports) == 1:
+            [port] = ports
+        else:
+            # TODO: Support multiple ports (issue #29)
+            port = sorted(ports)[0]
+            log.msg('Warning. %d ports found for app "%s". Consular currently '
+                    'only supports a single port. Only the lowest port (%s) '
+                    'will be used.' % (len(ports), app_id, port,))
```

Comment:
To be honest, I'm not really sure where the line between `msg` and `err` lies. If there were a `warn` the decision would be easy.